nextflow_schema.json

```diff
@@ -243,6 +255,42 @@
                 }
             }
         },
+        "tmb_calculation_options": {
+            "title": "TMB Calculation options",
+            "type": "object",
+            "description": "Options related to cutoffs in TMB calculation",
+            "default": "",
+            "properties": {
+                "calculate_tmb": {
+                    "type": "boolean",
+                    "description": "Enable TMB calculation"
+                },
+                "min_af": {
+                    "type": "number",
+                    "default": 0.0,
+                    "description": "Minimal allele frequency cutoff for TMB eligibility"
```

Comment:
maybe replace "eligibility" with "calculation" to emphasize this is a filter for the TMB calculation step.